models/l2-sea/README.md

```diff
@@ -3,25 +3,25 @@
 ## Overview
 The model provides the calm-water total resistance of a destroyer-type vessel as a function of the advancing speed (Froude number) and up to 14 design variables for the shape modification. The parent vessel under investigation is the DTMB 5415, an hull-form widley used for towing tank experiments, computational fluid dynamics studies, and shape optimization. The model is available as an open-source fortran code for the solution of potential flow equations at CNR-INM, MAO Research Group repository github.com/MAORG-CNR-INM/NATO-AVT-331-L2-Sea-Benchmark.   
 
+![L2-Sea-Model](https:/github.com/UM-Bridge/benchmarks/edit/main/models/l2-sea/figs/l2-sea_example.png "DTMB 5415 view of the wave elevation pattern and pressure field on the hull surface")
+
 ## Authors
-Andrea Serani, Lorenzo Tamellini, Riccardo Pellegrini, Matteo Diez
+- [Andrea Serani](mailto:andrea.serani@cnr.it)
+- [Lorenzo Tamellini](mailto:lorenzo.tamellini@cnr.it)
+- [Riccardo Pellegrini](mailto:riccardo.pellegrini@cnr.it)
+- [Matteo Diez](mailto:matteo.diez@cnr.it)
 
 ## Run
 ```
-docker run -it -p 4242:4242 linusseelinger/model-l2-sea
+docker run -it -p 4242:4242 -v ~/l2-sea_output:/output linusseelinger/model-l2-sea 
 ```
 
 ## Properties
 
 Model | Description
 ---|---
 forward | l2-sea
-
```

Comment:
Your model container (at least the one from the repo) provides a single model named "forward", so that should still be listed here as before, including documentation of its input/output dimension etc.

Since (I believe) the optimization benchmark makes use of the same forward model, it's fine to just define the optimization problem in the description .

Reply:
yes, that's fine. Would you like to directly make the correction?

Reply:
Sure, will do! We'll also take care of formatting of formulas